io.github.jliljebl.Flowblade.json

```diff
@@ -0,0 +1,539 @@
+                }
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig"
+            ]
+        },
+        {
+            "name": "pygobject",
+            "build-options": {
+                "env": {
+                    "PYTHON": "/usr/bin/python2"
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/pygobject/3.26/pygobject-3.26.1.tar.xz",
+                    "sha256": "f5577b9b9c70cabb9a60d81b855d488b767c66f867432e7fb64aa7269b04d1a9"
+                }
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig",
+                "/lib/python2.7/site-packages/gi/*.la"
+            ]
+        },
+        {
+            "name": "swig",
+            "config-opts": [
+                "--without-alllang",
+                "--with-python=/usr/bin/python",
+                "--without-boost",
+                "--without-pcre"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://sourceforge.net/projects/swig/files/swig/swig-3.0.12/swig-3.0.12.tar.gz",
+                    "sha256": "7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d"
+                }
+            ],
+            "cleanup": [
+                "*"
+            ]
+        },
+        {
+            "name": "SDL1",
+            "rm-configure": true,
+            "config-opts": [
+                "--disable-static",
+                "--disable-pulseaudio",
+                "--enable-alsa"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.libsdl.org/release/SDL-1.2.15.tar.gz",
+                    "sha256": "d6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00"
+                },
+                {
+                    "type": "patch",
+                    "path": "sdl-libx11-build.patch"
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "autogen.sh",
+                    "commands": [
+                        "sed -i -e 's/.*AM_PATH_ESD.*//' configure.in",
+                        "cp -p /usr/share/automake-*/config.{sub,guess} build-scripts",
+                        "aclocal",
+                        "libtoolize",
+                        "autoconf"
+                    ]
+                }
+            ],
+            "cleanup": [
+                "/bin",
+                "/share/man",
+                "/share/aclocal",
+                "/include",
+                "/lib/pkgconfig",
+                "/lib/*.la",
+                "/lib/*.a"
+            ]
+        },
+        {
+            "name": "jack2",
```

Comment:
If `jack` support is optional we should probably disable it. `pulseaudio` should be the only audio method out of the sandbox.

Reply:
I would like to let upstream decide that after this is accepted. The app needs `--filesystem=host` anyway, so safety is not much worse.